api/user.json

```diff
@@ -78,16 +96,53 @@
   ],
   "deductions": {
     "disabilityClaim": null,
+    "disabilityAmount": {
+      "name": "Disability",
+      "line": 316,
+      "amount": 0.00
+    },
     "rrspClaim": null,
+    "basicPersonalAmount": {
+      "name": "Basic Personal Amount",
+      "line": 300,
+      "amount": 11809.00
+    },
+    "ageAmount": {
+      "name": "Age Amount",
+      "line": 301,
+      "amount": 7500.00
+    },
     "rrspAmount": 0,
     "partnerAgeClaim": null,
     "canadaCaregiverClaim": null,
     "partnerDisabilityClaim": null,
     "partnerPensionIncomeClaim": null,
+    "pensionIncomeClaim": null,
+    "pensionIncomeClaimAmount": {
+      "name": "Pension",
+      "line": 314,
+      "amount": 700.00
+    },
     "childcareClaim": null,
+    "childcareAmount": {
+      "name": "Childcare Expenses",
+      "line": 214,
+      "amount": 1517.94
+    },
     "singleParentClaim": null,
     "dependentDisabilityClaim": null,
     "medicalExpenseClaim": null,
+    "medicalExpenseClaimAmount": {
+      "name": "Medical Expenses",
+      "line": 331,
+      "multiLine": "330/331",
+      "amount": 1227.34
```

Comment:
If @sboisvert's #69 PR goes in, you'll have to refactor the logic on his medical expense page and the POST route.